test/morgan.js

```diff
@@ -822,6 +822,79 @@ describe('morgan()', function () {
         test.write('0')
       })
     })
+
+    describe('a custom', function () {
+      function token (req, res, arg) {
+        return arg
+      }
+      var url
+      beforeEach(function () {
+        url = morgan.url
+        morgan.token('url', token)
```

Comment:
Hm. I've been thinking on the "unclear why I need different ones" and I feel like I described it above, but obviously we have a disconnect here.. not sure how to proceed. Thoughts?